scripts/contractInteraction/staking_vesting.py

```diff
@@ -366,11 +367,32 @@ def updateLockedSOV():
     # sendWithMultisig(conf.contracts['multisig'], lockedSOV.address, data, conf.acct)
 
 #receiver is usually the multisig
-def governanceWithdrawVesting( vesting,  receiver):
+def governanceDirectWithdrawVesting( vesting,  receiver):
     stakingProxy = Contract.from_abi("Staking", address=conf.contracts['Staking'], abi=Staking.abi, owner=conf.acct)
-    data = stakingProxy.governanceWithdrawVesting.encode_input( vesting,  receiver)
-    print(data)
-    sendWithMultisig(conf.contracts['multisig'], conf.contracts['Staking'], data, conf.acct)
+    vesting = Contract.from_abi("VestingLogic", address=vesting, abi=VestingLogic.abi, owner=conf.acct)
+
+    DAY = 24 * 60 * 60
+    FOUR_WEEKS = 4 * 7 * DAY
+
+    vestingStartDate = vesting.startDate()
+    vestingEnd = vesting.endDate()
+    vestingCliff = vesting.cliff()
+    defaultStart = vestingStartDate + vestingCliff
+
+    maxIterations = stakingProxy.getMaxVestingWithdrawIterations()
+    maxIterationsEnd = defaultStart + (FOUR_WEEKS * maxIterations)
+
+    counter = 1;
```

Comment:
```suggestion
    counter = 1
```

Reply:
Fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/c4c43c22e2dc30a936960e01e9a8acd6e2053616)